mmf/datasets/processors/text_image_processors.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
We also discourage new processors files until unless it is absolute necessary. So, please move this into one of the processor files already existing.

Reply:
Let me know if you want to know the reason behind this.